app/src/main/java/co/runloop/influencer/utils/SimpleObserver.java

```diff
@@ -0,0 +1,6 @@
+package co.runloop.influencer.utils;
+
+public interface SimpleObserver {
```

Comment:
Why not ? It have only 1 method and without data.
What a name do you think will be better ?

Reply:
It's just that simple isn't a well defined term. It's just a word... What if it had 2 methods? It could still be simple then  :)

I also wonder if it's a good idea to declare it in a utils package. 

I'm not sure what would be a better name. `DataObserver` maybe? 

What's is the exact role for this interface? If we were not using Java 8 and didn't have this lambda notation, how would this relationship between the contacts provider and the view model work? Some callback interface?

Reply:
Idea of SimpleObserver - to observe changes in contacts i need to implement very heavy observer, but it not needed in my ui, because it does not have useful info. So heavy observer implemented in provider, and ui needs just a simple trigger to know when data is updated

Reply:
Ok, so - "simple" isn't descriptive of it's purpose. When we name entities in the code, their name should make it clear what their purpose is. When you see something like "SimpleObserver", you can't really tell what it does without inspecting it... Do you understand what I mean?